kairos-cli/bin/common/args.rs

```diff
@@ -0,0 +1,15 @@
+use std::path::PathBuf;
+
+use clap::Args;
+
+#[derive(Args, Debug)]
+pub struct AmountArg {
+    #[arg(name = "amount", long, short, value_name = "NUM_MOTES")]
+    pub field: u64,
+}
+
+#[derive(Args, Debug)]
+pub struct PrivateKeyPathArg {
+    #[arg(name = "private-key", long, short = 'k', value_name = "FILE_PATH")]
```

Comment:
can we use value_parse here to immediately parse this to a private-key?

Reply:
An interesting read on this btw: https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/

Reply:
I tried that... but it appeared that `Clone` must be implemented for the argument, which is not present in Casper types:

```
error[E0277]: the trait bound `casper_types::SecretKey: std::clone::Clone` is not satisfied
 --> kairos-cli/bin/crypto/private_key.rs:6:29
  |
5 | #[derive(Clone)]
  |          ----- in this derive macro expansion
6 | pub struct CasperPrivateKey(pub casper_types::SecretKey);
  |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::clone::Clone` is not implemented for `casper_types::SecretKey`